src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MitochondrialFiltersArgumentCollection.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect;
+
+import org.broadinstitute.barclay.argparser.Argument;
+
+
+public class MitochondrialFiltersArgumentCollection extends M2FiltersArgumentCollection {
```

Comment:
Actually maybe not.  Now that I look again, this is going to give the MitochondrialFiltersArgumentCollection a `tumor-segmentation` argument, which I don't want.

Reply:
Oh you're right, and now that I look at it, M2FiltersArgumentCollection extends AssemblyBasedCallerArgumentCollection, which doesn't make sense to me because those shouldn't be used in the filtering step (eg `bamout`). 

But assuming that does make sense, does that mean I want the MitochondrialFiltersArgumentCollection to extend AssemblyBasedCallerArgumentCollection?

Reply:
It doesn't look to me like FilterMutectCalls needs any of the AssemblyBasedCaller args either.  There could be a FiltersArgumentCollection that's the parent of an M2 version and a mito version.  I think that arg collection wouldn't inherit from anyone.

Reply:
It looks like there's going to be a lot of duplication just because tumors and mitochondria need different defaults.  Is there really no elegant way to do this?